code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -494,6 +494,27 @@
 
 	reagents_to_add = list(/decl/reagent/nutriment = 10, /decl/reagent/sugar = 3)
 
+/obj/item/reagent_containers/food/snacks/proteinbar
+	name = "protein bar"
+	desc = "SwoleMAX brand protein bars, guaranteed to get you feeling perfectly overconfident."
```

Comment:
```suggestion
	desc = "Getmore brand protein bars, guaranteed to get you feeling perfectly overconfident."
```
still unaddressed.

Reply:
lampshade it as a subsidiary of Getmore

Reply:
or change it to Getmore as suggested and no "lampshading" has to be done.